spec/system/admin/products_v3/index_spec.rb

```diff
@@ -91,6 +75,38 @@
           expect(page).to have_content "4"
         end
       end
+
+      describe "Producer column" do
+        it "when I have one enterprise" do
+          visit spree.admin_products_path
+
+          expect(page).to_not have_select "Producer"
+        end
+
+        context "when I have multiple enterprises" do
+          let(:user) { create(:user, enterprises: [producer2, producer1]) }
+          let(:producer1) { create(:supplier_enterprise, name: "Producer A") }
+          let(:producer2) { create(:supplier_enterprise, name: "Producer B") }
+
+          it "displays a select box for suppliers, with the appropriate supplier selected" do
+            producer3 = create(:supplier_enterprise, name: "Producer C")
```

Comment:
I am surprise rubocop is not telling you `producer_3` is not used. No big deal anyway.

Reply:
🙈 
```suggestion
           create(:supplier_enterprise, name: "Producer C")
```